ykrt/src/compile/jitc_yk/codegen/x64/mod.rs

```diff
@@ -139,10 +139,17 @@ pub(crate) type VarLocation = super::reg_alloc::VarLocation<Register>;
 
 /// The lock used when patching a side-trace into a parent trace.
 static LK_PATCH: Mutex<()> = Mutex::new(());
-/// x86 JMP opcode.
+/// x86 JMP opcode for a 32bit offset.
 const JMP_OPCODE: u8 = 0xE9;
 /// x86 NOP opcode.
-const NOP_OPCODE: u8 = 0x80;
+const NOP_OPCODE: u8 = 0x90;
+/// x64 opcode to jump to an address stored in [RSP-8].
+const JMP_RSP_8: u32 = 0xFF6424F8;
+/// x64 opcode to move an imm32 into [rsp-8].
```

Comment:
I assume this is meant to be `imm64`?

Reply:
No, we need to split the 64-bit address up into two 32-bit parts and write one to `rsp-8` and the other to `rsp-4`.

Reply:
OK can we document that these are two halfs of a 64 bit address and we have to write them in two parts?

Reply:
This is also gone now.